tools/travis/docker_run.sh

```diff
@@ -11,6 +11,12 @@ eval "$(pyenv init -)"
 pyenv global ${PYVER}
 pyenv local ${PYVER}
 
+gcc -v
+
+git log -10
+
+swig -version
```

Comment:
To be removed ?

Reply:
sure, I think it's useful for debugging tho

Reply:
Ok, as you told me we better keep those :)